src/main/java/laura/UI.java

```diff
@@ -0,0 +1,34 @@
+package laura;
+
+/**
+ * For displaying messages for the user to see
+ */
+public class UI {
+    /**
+     * Sends the Welcome Message for when the application begins
+     */
+    public static void welcomeMessage() {
+        UI.send(
+                "Hello I'm Laura\n" +
+                        "What can I do for you?"
```

Comment:
Similar to other parts, perhaps a line break before the + operator?
```suggestion
                "Hello I'm Laura\n"
                + "What can I do for you?"
```